src/pages/Popup/components/timer.jsx

```diff
@@ -0,0 +1,164 @@
+import React, { Component } from "react";
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+
+const timerStyle = {
```

Comment:
Why have you defined styles two times? Over here and at https://github.com/devlup-labs/meet-scheduler/pull/78/files#diff-d68e4e1cfe5380202759d6f1e773e5f8a580722a85d79de1daa002b98e13afe6R21

But it seems you are not even using them because you have defined inline styles at https://github.com/devlup-labs/meet-scheduler/pull/78/files#diff-d68e4e1cfe5380202759d6f1e773e5f8a580722a85d79de1daa002b98e13afe6R149

Reply:
it was an experimental piece of code but has been removed